generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -63,23 +61,23 @@ class WithScalaTestFeatures extends Config((site, here, up) => {
   case TracePortKey => up(TracePortKey, site).map(_.copy(print = true))
 })
 
-
 // FASED Config Aliases. This to enable config generation via "_" concatenation
 // which requires that all config classes be defined in the same package
 class DDR3FRFCFS extends FRFCFS16GBQuadRank
 class DDR3FRFCFSLLC4MB extends FRFCFS16GBQuadRankLLC4MB
 
 class WithNIC extends icenet.WithIceNIC(inBufFlits = 8192, ctrlQueueDepth = 64)
 
-
-
 // Enables tracing on all cores
 class WithTraceIO extends Config((site, here, up) => {
   case BoomTilesKey => up(BoomTilesKey) map (tile => tile.copy(trace = true))
   case ArianeTilesKey => up(ArianeTilesKey) map (tile => tile.copy(trace = true))
   case TracePortKey => Some(TracePortParams())
 })
 
+// Adds a small/large NVDLA to the system
+class WithNVDLALarge extends nvidia.blocks.dla.WithNVDLA("large")
+class WithNVDLASmall extends nvidia.blocks.dla.WithNVDLA("small")
```

Comment:
Why are these here?

Reply:
IIRC to do something like this `WithNVDLASmall_DDR3FRFCFSLLC4MB_FireSimQuadRocketConfig` in the build recipe you need to have `WithNVDLA...` here. (so that everything is in the same scala package). I assume that this would be the easiest way to add an NVDLA for external users.

Reply:
See https://github.com/ucb-bar/nvdla-wrapper/blob/master/firesim-collateral/nvdla_config_build_recipes.ini.